### ISSUE TITLE
hls-class-plugin setting fields are missing

### ISSUE THREAD

**Issue by July541** [haskell/vscode-haskell#686]
After https://github.com/haskell/haskell-language-server/pull/2920, hls-class-plugin has two configurable fields now(after https://github.com/haskell/haskell-language-server/pull/3107), including:

1. `haskell.plugin.class.codeActionsOn`
2. `haskell.plugin.class.codeLensOn`

But currently vscode-haskell only has the first one.

**Comment by fendor** [comment]
Weird, must have slipped through :(

**Comment by July541** [comment]
My fault, I should raise an issue here after https://github.com/haskell/haskell-language-server/pull/2920. But the pr takes so much time, I forgot it...

**Comment by July541** [comment]
BTW, will you update the fields by scanning the pr list in hls or rely on `haskell-language-server generate-default-config`?

**Comment by fendor** [comment]
I rely on `haskell-language-server vscode-extension-schema` which I copy-paste into package.json. More automation would be welcome, so far I couldn't find any existing infrastructure for doing it automatically.

However, I am currently working on printing the json with sorted keys, making sure we haven't deleted anything on accident and making it easier to view added keys.

**Comment by fendor** [comment]
I think this will be fixed in #773, @July541 could you make sure the current version of package.json contains all the hls-class-plugin options it should contain?

**Comment by July541** [comment]
Looks reasonable to me, and I confirmed

```json
{
    "haskell.plugin.class.codeLensOn": false,
    "haskell.plugin.class.codeActionsOn": false
}
```

works for me.

Let's go!
